wallet/test/unit_testing/services/repository/repository_offline_test.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:fixnum/fixnum.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:http/http.dart' as http;
+import 'package:pylons_wallet/model/stripe_account_link_request.dart';
+import 'package:pylons_wallet/model/stripe_account_link_response.dart';
+import 'package:pylons_wallet/model/stripe_create_payment_intent_request.dart';
+import 'package:pylons_wallet/model/stripe_create_payment_intent_response.dart';
+import 'package:pylons_wallet/model/stripe_generate_payment_receipt_request.dart';
+import 'package:pylons_wallet/model/stripe_generate_payment_receipt_response.dart';
+import 'package:pylons_wallet/model/stripe_generate_payout_token_request.dart';
+import 'package:pylons_wallet/model/stripe_generate_payout_token_response.dart';
+import 'package:pylons_wallet/model/stripe_generate_registration_token_response.dart';
+import 'package:pylons_wallet/model/stripe_generate_update_token_response.dart';
+import 'package:pylons_wallet/model/stripe_get_login_based_address.dart';
+import 'package:pylons_wallet/model/stripe_payout_request.dart';
+import 'package:pylons_wallet/model/stripe_payout_response.dart';
+import 'package:pylons_wallet/model/stripe_register_account_response.dart';
+import 'package:pylons_wallet/model/stripe_register_acount_request.dart';
+import 'package:pylons_wallet/model/stripe_update_account_request.dart';
+import 'package:pylons_wallet/model/stripe_update_account_response.dart';
+import 'package:pylons_wallet/services/repository/repository.dart';
+import 'package:pylons_wallet/stores/wallet_store.dart';
+import 'package:pylons_wallet/utils/backup/common/i_driver_client.dart';
+
+import '../../../mocks/mock_constants.dart';
+import '../../../mocks/mock_crashlytics_helper.dart';
+import '../../../mocks/mock_google_drive_helper.dart';
+import '../../../mocks/mock_http_client.dart';
+import '../../../mocks/mock_icloud_drive_helper.dart';
+import '../../../mocks/mock_local_auth_helper.dart';
+import '../../../mocks/mock_local_data_source.dart';
+import '../../../mocks/mock_network_info_offline.dart';
+import '../../../mocks/mock_query_helper.dart';
+import '../../../mocks/mock_remote_data_store.dart';
+import '../../../mocks/mock_wallet_store.dart';
+
+void main() async {
+  dotenv.testLoad(fileInput: '''ENV=true''');
```

Comment:
I dont think unit test needs dotenv.

Reply:
I tried removing it. But now we are using `MOCK_BASE_ENV`, which uses `setEnv `function and that function uses this check `(dotenv.env[kENV]! == kLocal)` .